moveit_core/collision_detection_fcl/test/test_fcl_collision_detection.cpp

```diff
@@ -286,10 +286,15 @@ TEST_F(FclCollisionDetectionTester, AttachedBodyTester)
   crobot_->checkSelfCollision(req, res, robot_state, *acm_);
   ASSERT_FALSE(res.collision);
 
-  shapes::Shape* shape = new shapes::Box(.1, .1, .1);
+  shapes::Shape* shape = new shapes::Box(.2, .2, .2);
```

Comment:
What's the reason for this change?

Reply:
I think Bullet wraps around its objects an additional contact distance threshold, therefore I increased the Box size in the FCL test to get the same resulting collisions. If I don't increase it for FCL, Bullet reports 4 collision whereas FCL only 2.

Reply:
Interesting! I found [this bullet forum post](https://pybullet.org/Bullet/phpBB3/viewtopic.php?t=6575#p22856), which says the default "margin" is 0.04m, which would probably cause the same number of contacts to be reported as expanding the box by 0.1m. This margin will be interesting to tinker with in the coming weeks.

Reply:
![image](https://user-images.githubusercontent.com/19279526/58482765-4b7efb00-815f-11e9-8a66-87598254c217.png)

Reply:
distance threshold is 0.05, I am not sure where exactly this is set yet.